templates/etc/ssh/sshd_config.j2

```diff
@@ -17,8 +17,6 @@ Protocol 2
 {% for hostkey in ssh_host_keys %}
 HostKey {{ hostkey }}
 {% endfor %}
-#Privilege Separation is turned on for security
```

Comment:
Why remove the `UsePrivilegeSeparation yes`? From my point it's a good thing? Or is this not supported anymore on SLES15?

Reply:
I should not review before coffee and reading the whole description. LGTM